Source/Scene/Model/MetadataPipelineStage.js

```diff
@@ -15,31 +15,52 @@ import ModelUtility from "./ModelUtility.js";
  *
  * @private
  */
-const MetadataPipelineStage = {};
-MetadataPipelineStage.name = "MetadataPipelineStage";
-
-MetadataPipelineStage.STRUCT_ID_METADATA_VS = "MetadataVS";
-MetadataPipelineStage.STRUCT_ID_METADATA_FS = "MetadataFS";
-MetadataPipelineStage.STRUCT_NAME_METADATA = "Metadata";
-MetadataPipelineStage.STRUCT_ID_METADATACLASS_VS = "MetadataClassVS";
-MetadataPipelineStage.STRUCT_ID_METADATACLASS_FS = "MetadataClassFS";
-MetadataPipelineStage.STRUCT_NAME_METADATACLASS = "MetadataClass";
-MetadataPipelineStage.FUNCTION_ID_INITIALIZE_METADATA_VS =
-  "initializeMetadataVS";
-MetadataPipelineStage.FUNCTION_ID_INITIALIZE_METADATA_FS =
-  "initializeMetadataFS";
-MetadataPipelineStage.FUNCTION_SIGNATURE_INITIALIZE_METADATA =
-  "void initializeMetadata(out Metadata metadata, out MetadataClass metadataClass, ProcessedAttributes attributes)";
-MetadataPipelineStage.FUNCTION_ID_SET_METADATA_VARYINGS = "setMetadataVaryings";
-MetadataPipelineStage.FUNCTION_SIGNATURE_SET_METADATA_VARYINGS =
-  "void setMetadataVaryings()";
-// Metadata class info: some fields must be renamed to avoid reserved words
-MetadataPipelineStage.METADATACLASS_FIELDS = [
-  { specName: "noData", shaderName: "noData" },
-  { specName: "default", shaderName: "defaultValue" },
-  { specName: "min", shaderName: "minValue" },
-  { specName: "max", shaderName: "maxValue" },
-];
+const MetadataPipelineStage = {
+  name: "MetadataPipelineStage",
+
+  STRUCT_ID_METADATA_VS: "MetadataVS",
+  STRUCT_ID_METADATA_FS: "MetadataFS",
+  STRUCT_NAME_METADATA: "Metadata",
+
+  STRUCT_ID_METADATACLASS_VS: "MetadataClassVS",
+  STRUCT_ID_METADATACLASS_FS: "MetadataClassFS",
+  STRUCT_NAME_METADATACLASS: "MetadataClass",
+
+  STRUCT_ID_METADATASTATISTICS_VS: "MetadataStatisticsVS",
```

Comment:
nitpick: even though the class is called `MetadataStatistics`, I think `METADATA_STATISTICS` would be more readable for the constant names. Similar for `METADATA_CLASS`